cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -66,7 +67,8 @@ class PredicateFunctionsTest extends DocumentingTest {
       }
     }
     section("any()", "functions-any") {
-      p("`any()` returns true if the predicate holds for at least one element in the given list.")
+      p("`any()` returns true if the predicate holds for at least one element in the given list." +
+        " Returns `null` if the list is `null` or all of its elements are `null`.")
```

Comment:
"Returns `null` if the..." -> "`null` is returned if the ..."  (to make it a sentence, rather than a clause)